.github/workflows/ci.yml

```diff
@@ -0,0 +1,31 @@
+name: CI
```

Comment:
We should change the name for this file, as well as the `name` value here, since it's a pipeline specific to the keychain.

Reply:
It's meant to be general purpose CI, though the starting point is the OSX tests. It just so happens that the only OSX thing running now is the keychain tests.

Reply:
I've uncommented the build job so that it's now general purpose.